packages/web/docs/src/content/gateway/other-features/security/disable-introspection.mdx

```diff
@@ -44,46 +54,3 @@ export const gatewayConfig = defineConfig({
   }
 })
 ```
-
-## Disabling Field Suggestions
```

Comment:
Isn't this important to have this here? Because this plugin prevents the leak of the introspection even if it is disabled. So user will know it can still leak.

Reply:
It's still there, just on another, dedicated, page: https://pr-6630.hive-landing-page.pages.dev/docs/gateway/other-features/security/block-field-suggestions.

Reply:
BTW the "block field suggestions" has nothing to do with introspection. It's the suggestion GQL generates in an error when you mistype a field.

```
Cannot query field "upcie" on type "Product". Did you mean "upc"?
```

->

```
Cannot query field "upcie" on type "Product". [Suggestion hidden]
```

Reply:
`Did you mean X?` exposes the parts of introspection. And theoritically it is possible to get the schema fully even if the introspection is disabled. I'm trying to find that tool that creates the entire schema for you by brute forcing this suggestion thing. So I think it is related. The point of disabling introspection is to prevent your schema to leak. So user should know this is not enough which is a common mistake.

Reply:
See https://github.com/nikitastupin/clairvoyance?ref=escape.tech

Reply:
https://github.com/graphql-hive/console/pull/6630/commits/d8ae11210012a8a85a226632391cec2de4ceacdd